De-fuzz the patch that the poedit build applies to its copy of gettext
to appease the picky patch program in macOS 13.
https://github.com/vslavik/poedit/issues/781
--- deps/gettext/gettext-macos-catalina-dont-touch-parent-dirs.patch.orig	2023-02-07 12:25:36.000000000 -0600
+++ deps/gettext/gettext-macos-catalina-dont-touch-parent-dirs.patch	2023-02-07 12:21:54.000000000 -0600
@@ -3,48 +3,48 @@
 happens to be in e.g. ~/Desktop or ~/Downloads. As we don't care for these
 capabilities in gettext tools anyway, just disable them as the lesser evil.
 
-diff -ru gettext-0.20.2.orig/gettext-runtime/configure gettext-0.20.2/gettext-runtime/configure
---- gettext-0.20.2.orig/gettext-runtime/configure	2020-04-14 06:45:37.000000000 +0200
-+++ gettext-0.20.2/gettext-runtime/configure	2020-05-07 19:27:29.000000000 +0200
-@@ -20727,7 +20727,7 @@
+diff -ru gettext-0.21.1.orig/gettext-runtime/configure gettext-0.21.1/gettext-runtime/configure
+--- gettext-0.21.1.orig/gettext-runtime/configure	2022-10-09 16:31:34.000000000 -0500
++++ gettext-0.21.1/gettext-runtime/configure	2023-02-07 12:18:37.000000000 -0600
+@@ -25602,7 +25602,7 @@
       LIBS="$LIBS -Wl,-framework -Wl,CoreFoundation"
       cat confdefs.h - <<_ACEOF >conftest.$ac_ext
  /* end confdefs.h.  */
 -#include <CoreFoundation/CFPreferences.h>
 +#include <Disabled_CoreFoundation/CFPreferences.h>
  int
- main ()
+ main (void)
  {
-@@ -20761,7 +20761,7 @@
+@@ -25638,7 +25638,7 @@
       LIBS="$LIBS -Wl,-framework -Wl,CoreFoundation"
       cat confdefs.h - <<_ACEOF >conftest.$ac_ext
  /* end confdefs.h.  */
 -#include <CoreFoundation/CFLocale.h>
 +#include <Disabled_CoreFoundation/CFLocale.h>
  int
- main ()
+ main (void)
  {
-@@ -21184,7 +21184,7 @@
+@@ -26149,7 +26149,7 @@
       LIBS="$LIBS -Wl,-framework -Wl,CoreFoundation"
       cat confdefs.h - <<_ACEOF >conftest.$ac_ext
  /* end confdefs.h.  */
 -#include <CoreFoundation/CFPreferences.h>
 +#include <Disabled_CoreFoundation/CFPreferences.h>
  int
- main ()
+ main (void)
  {
-@@ -21218,7 +21218,7 @@
+@@ -26185,7 +26185,7 @@
       LIBS="$LIBS -Wl,-framework -Wl,CoreFoundation"
       cat confdefs.h - <<_ACEOF >conftest.$ac_ext
  /* end confdefs.h.  */
 -#include <CoreFoundation/CFLocale.h>
 +#include <Disabled_CoreFoundation/CFLocale.h>
  int
- main ()
+ main (void)
  {
-diff -ru gettext-0.20.2.orig/gettext-runtime/m4/intlmacosx.m4 gettext-0.20.2/gettext-runtime/m4/intlmacosx.m4
---- gettext-0.20.2.orig/gettext-runtime/m4/intlmacosx.m4	2020-04-13 10:42:28.000000000 +0200
-+++ gettext-0.20.2/gettext-runtime/m4/intlmacosx.m4	2020-05-07 19:27:29.000000000 +0200
+diff -ru gettext-0.21.1.orig/gettext-runtime/m4/intlmacosx.m4 gettext-0.21.1/gettext-runtime/m4/intlmacosx.m4
+--- gettext-0.21.1.orig/gettext-runtime/m4/intlmacosx.m4	2022-10-08 21:35:42.000000000 -0500
++++ gettext-0.21.1/gettext-runtime/m4/intlmacosx.m4	2023-02-07 12:18:37.000000000 -0600
 @@ -24,7 +24,7 @@
       LIBS="$LIBS -Wl,-framework -Wl,CoreFoundation"
       AC_LINK_IFELSE(
@@ -63,42 +63,42 @@
            [[CFLocaleCopyPreferredLanguages();]])],
         [gt_cv_func_CFLocaleCopyPreferredLanguages=yes],
         [gt_cv_func_CFLocaleCopyPreferredLanguages=no])
-diff -ru gettext-0.20.2.orig/gettext-tools/configure gettext-0.20.2/gettext-tools/configure
---- gettext-0.20.2.orig/gettext-tools/configure	2020-04-14 03:34:46.000000000 +0200
-+++ gettext-0.20.2/gettext-tools/configure	2020-05-07 19:27:29.000000000 +0200
-@@ -23270,7 +23270,7 @@
+diff -ru gettext-0.21.1.orig/gettext-tools/configure gettext-0.21.1/gettext-tools/configure
+--- gettext-0.21.1.orig/gettext-tools/configure	2022-10-09 16:32:05.000000000 -0500
++++ gettext-0.21.1/gettext-tools/configure	2023-02-07 12:18:37.000000000 -0600
+@@ -28356,7 +28356,7 @@
       LIBS="$LIBS -Wl,-framework -Wl,CoreFoundation"
       cat confdefs.h - <<_ACEOF >conftest.$ac_ext
  /* end confdefs.h.  */
 -#include <CoreFoundation/CFPreferences.h>
 +#include <Disabled_CoreFoundation/CFPreferences.h>
  int
- main ()
+ main (void)
  {
-@@ -23304,7 +23304,7 @@
+@@ -28392,7 +28392,7 @@
       LIBS="$LIBS -Wl,-framework -Wl,CoreFoundation"
       cat confdefs.h - <<_ACEOF >conftest.$ac_ext
  /* end confdefs.h.  */
 -#include <CoreFoundation/CFLocale.h>
 +#include <Disabled_CoreFoundation/CFLocale.h>
  int
- main ()
+ main (void)
  {
-@@ -23727,7 +23727,7 @@
+@@ -28903,7 +28903,7 @@
       LIBS="$LIBS -Wl,-framework -Wl,CoreFoundation"
       cat confdefs.h - <<_ACEOF >conftest.$ac_ext
  /* end confdefs.h.  */
 -#include <CoreFoundation/CFPreferences.h>
 +#include <Disabled_CoreFoundation/CFPreferences.h>
  int
- main ()
+ main (void)
  {
-@@ -23761,7 +23761,7 @@
+@@ -28939,7 +28939,7 @@
       LIBS="$LIBS -Wl,-framework -Wl,CoreFoundation"
       cat confdefs.h - <<_ACEOF >conftest.$ac_ext
  /* end confdefs.h.  */
 -#include <CoreFoundation/CFLocale.h>
 +#include <Disabled_CoreFoundation/CFLocale.h>
  int
- main ()
+ main (void)
  {
